slides/src/content/a11y.md

```diff
@@ -282,6 +282,8 @@ Example HTML for a table:
 
 - Use headers in sequnce (H1 - H6)
 
+- Note, if you are using bootstrap templates, you are likely going to have to implement accessibility in detail as webkits don't have out of the box support for it
```

Comment:
- 'Bootstrap' instead of 'bootstrap'
- what are "webkits"
- remove "likely" => we're here to tell them...